.github/workflows/kbs-client-build-and-push.yaml

```diff
@@ -0,0 +1,39 @@
+name: Build and push kbs-client
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build_and_push:
+    env:
+      RUSTC_VERSION: 1.76.0
+    runs-on: ubuntu-22.04
+
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v4
+    - name: Install rust toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: ${{ env.RUSTC_VERSION }}
+        override: true
+        profile: minimal
+    - name: Log in to ghcr.io
+      uses: docker/login-action@v3
+      with:
+        registry: ghcr.io
+        username: ${{ github.actor }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Build a statically linked kbs-client for x86_64 linux
+      working-directory: kbs
+      run: |
+        make cli-static-x86_64-linux
+    - name: Push to ghcr.io
+      working-directory: target/x86_64-unknown-linux-gnu/release
+      run: |
+        commit_sha=${{ github.sha }}
+        oras push \
```

Comment:
Is oras installed already?

Reply:
good question.

Reply:
It should be there. I tested it was there, but also a github action runner from this PR says its installed software is [here](https://github.com/confidential-containers/trustee/actions/runs/8266650115/job/22615127521?pr=349#step:1:10) (see [ORAS under cli tools](https://github.com/actions/runner-images/blob/ubuntu22/20240304.1/images/ubuntu/Ubuntu2204-Readme.md#cli-tools)).

Reply:
Ah I guess they are adding stuff on top of Ubuntu